rules/0195-named_rules.xml

```diff
@@ -329,6 +335,10 @@
     <if_matched_sid>12108</if_matched_sid>
     <same_source_ip />
     <description> Multiple query (cache) failures.</description>
+    <mitre>
+      <id>T1518</id>
```

Comment:
This technique talks about software discovery (installed on the system). This rule is about DNS queries.